Zend/tests/036.phpt

```diff
@@ -11,4 +11,4 @@ try {
 
 ?>
 --EXPECT--
-Illegal offset type
+Illegal offset type: cannot be of type object
```

Comment:
Illegal array offset type of `object` would be shorter and sound better

Reply:
Thanks for the idea, it really gets better.

I defaulted to zend_illegal_string_offset, but that's up for debate.